ui/apps/platform/webpack.ocp-plugin.config.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
+const { ConsoleRemotePlugin } = require('@openshift-console/dynamic-plugin-sdk-webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const config = {
+    mode: isProd ? 'production' : 'development',
+    // No regular entry points needed. All plugin related scripts are generated via ConsoleRemotePlugin.
+    entry: {},
+    context: path.resolve(__dirname, 'src'),
+    output: {
+        path: path.resolve(__dirname, 'build', 'static', 'ocp-plugin'),
+        filename: isProd ? '[name]-bundle-[hash].min.js' : '[name]-bundle.js',
+        chunkFilename: isProd ? '[name]-chunk-[chunkhash].min.js' : '[name]-chunk.js',
+    },
+    resolve: {
+        extensions: ['.ts', '.tsx', '.js', '.jsx'],
+        alias: {
+            Containers: path.resolve(__dirname, 'src/Containers'),
+            Components: path.resolve(__dirname, 'src/Components'),
+            services: path.resolve(__dirname, 'src/services'),
+            utils: path.resolve(__dirname, 'src/utils'),
+            hooks: path.resolve(__dirname, 'src/hooks'),
+            types: path.resolve(__dirname, 'src/types'),
+            constants: path.resolve(__dirname, 'src/constants'),
+            queries: path.resolve(__dirname, 'src/queries'),
+            reducers: path.resolve(__dirname, 'src/reducers'),
+            sagas: path.resolve(__dirname, 'src/sagas'),
+            messages: path.resolve(__dirname, 'src/messages'),
+            mockData: path.resolve(__dirname, 'src/mockData'),
+            sorters: path.resolve(__dirname, 'src/sorters'),
+            'test-utils': path.resolve(__dirname, 'src/test-utils'),
+            images: path.resolve(__dirname, 'src/images'),
+            css: path.resolve(__dirname, 'src/css'),
+        },
```

Comment:
When I looked for corresponding aliases in vite config, I noticed helper function:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/vite.config.js#L48-L67

Is something similar relevant in this context?

Reply:
Yeah that is fair, I could create a similar helper function in this config. I'm hesitant to extract both to a single location though, as I don't know if there is a logical place to put helpers for separate build systems at this point. 🤔 

I'll clean this up in a follow up as well.